app/MMB/src/scenes/tripServices/insurance/InsuranceOverview.js

```diff
@@ -0,0 +1,44 @@
+// @flow strict
+
+import * as React from 'react';
+import { LayoutSingleColumn } from '@kiwicom/mobile-shared';
```

Comment:
Not strictly related to this PR, but I am not sure in my own Timeline view I use this `LayoutSingleColumn`. Should I?

Reply:
You should probably just align your container to the center.